app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,9 @@
   <body>
     <div class = "container">
     <%= render partial: "sessions/nav" %>
+    <center>
```

Comment:
@matthewhuff89  <center> is a depreciated tag, just get rid of it. Try to do it with CSS: text-align center.